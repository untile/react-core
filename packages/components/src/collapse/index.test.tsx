/**
 * Module dependencies.
 */

import { Collapse } from './';
import { render } from '@testing-library/react';
import React from 'react';

/**
 * Test `Collapse` component.
 */

describe(`'Collapse' component`, () => {
  it('matches snapshot when closed', () => {
    const { asFragment } = render(<Collapse>{'content'}</Collapse>);

    expect(asFragment()).toMatchSnapshot();
  });

  it('matches snapshot when opened', () => {
    const { asFragment } = render(<Collapse isOpen>{'content'}</Collapse>);

    expect(asFragment()).toMatchSnapshot();
  });
});
