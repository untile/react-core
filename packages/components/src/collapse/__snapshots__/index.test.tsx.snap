// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'Collapse' component matches snapshot when closed 1`] = `
<DocumentFragment>
  <div
    aria-hidden="true"
    class="rah-static rah-static--height-zero "
    style="height: 0px; overflow: hidden;"
  >
    <div
      style="display: none;"
    >
      content
    </div>
  </div>
</DocumentFragment>
`;

exports[`'Collapse' component matches snapshot when opened 1`] = `
<DocumentFragment>
  <div
    aria-hidden="false"
    class="rah-static rah-static--height-auto "
    style="height: auto; overflow: visible;"
  >
    <div>
      content
    </div>
  </div>
</DocumentFragment>
`;
